style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  font-family: sans-serif;
+  padding-left: 1%;
+}
+
+body {
+  width: 375px;
+  height: 718px;
+}
+
+.menu_line {
+  width: 35px;
+  height: 5px;
+  background-color: #6070ff;
+  margin: 6px 0;
+}
+
+.logo {
+  color: #6070ff;
+  font-style: italic;
```

Comment:
:red_circle:  Please make sure the logo text reflects the Figma design. Kindly remove the font-style to match the Figma Design

Your Design
![notthere](https://user-images.githubusercontent.com/26736582/205442587-e2786cba-1dfa-4e53-a41f-98145f431752.png)

Figma Design
<img src="https://github.com/microverseinc/curriculum-html-css/raw/main/portfolio/images/m1_setup/toolbar-mobile.png">